src/rtutils.c

```diff
@@ -641,6 +641,32 @@ static size_t jl_static_show_x_sym_escaped(JL_STREAM *out, jl_sym_t *name) JL_NO
     return n;
 }
 
+// `jl_static_show()` cannot call `jl_subtype()`, for the GC reasons
+// explained in the comment on `jl_static_show_x_()`, below.
+// This function checks if `vt <: Function` without triggering GC.
+static int jl_static_is_function_(jl_datatype_t *vt) JL_NOTSAFEPOINT {
+    if (!jl_function_type) {  // Make sure there's a Function type defined.
+        return 0;
+    }
+    int _iter_count = 0;  // To prevent infinite loops from corrupt type objects.
+    while (vt != jl_any_type) {
+        if (vt == NULL) {
+            jl_printf((JL_STREAM*)STDERR_FILENO,
+                "ERROR: static_show: Nullptr encountered inside datatype.\n");
```

Comment:
This isn't even an error, just the appearance of the #undef value. We format it as `#<null>` usually.

Reply:
Thanks - removed this, per the above.